### PR TITLE
Remove content from error msg, it should not be shown to user.

### PR DESCRIPTION
**Pull request by reiterl**
I changed the exception error msg, which shows the content. I just remove the content part.

### CODE REVIEW

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Please log the error instead of simply removing it, so the info is accessible in the backend log. Also, the more important place to update is in `handle_datastore_errors.py` where the actual errors of the datastore are handled.

**Comment by reiterl** [comment]
I have added a log msg.
What part of `handle_datastore_errors.py` should be changed?

File: openslides_backend/services/datastore/adapter.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def retrieve(self, command: commands.Command) -> Any:
                 payload = json.loads(content)
             except JSONDecodeError:
                 error_message = "Bad response from datastore service. Body does not contain valid JSON."
+                self.logger.debug(error_message + f" Received: {str(content)}")
```
Comment (jsangmeister) [inline openslides_backend/services/datastore/adapter.py:66]: This can be logged as an error, since the datastore should always send valid JSON.

**Comment by jsangmeister** [comment]
> What part of `handle_datastore_errors.py` should be changed?

The errors raised in `raise_datastore_error` should not contain the generated error message, analogous to this, but it should be logged on `debug` level

**Comment by reiterl** [comment]
Which one is the "generated error message"? How should this be logged, without logger?

**Comment by jsangmeister** [comment]
> Which one is the "generated error message"? How should this be logged, without logger?

Well, the messages passed into the exceptions, which there are two of. You will have to pass in a logger to log it.

File: openslides_backend/services/datastore/adapter.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/services/datastore/adapter.py]: Done.

File: openslides_backend/services/datastore/handle_datastore_errors.py [hunk 2]
```diff
@@ -36,21 +39,28 @@ def raise_datastore_error(
...
         elif type_verbose == "MODEL_DOES_NOT_EXIST":
             error_message = " ".join(
                 (
                     error_message,
                     f"Model '{additional_error_message.get('fqid')}' does not exist.",
                 )
             )
+            if logger:
+                logger.debug(error_message)
+            raise DatastoreException(error_message)
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/services/datastore/handle_datastore_errors.py:61]: Removed.

File: tests/system/action/speaker/test_create.py [hunk 3]
```diff
@@ -189,7 +189,7 @@ def test_create_add_2_speakers_in_2_actions(self) -> None:
         )
         self.assert_status_code(response, 400)
         self.assertIn(
-            "Datastore service sends HTTP 400. The following locks were broken: 'speaker/list_of_speakers_id', 'speaker/meeting_id', 'speaker/weight'",
+            "Datastore Error",
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/speaker/test_create.py:192]: Yes, there are a lot of checks in the tests. Including model missing tests.
I check "is_dev_mode" before I simplify the error_message, so the tests should pass. And I updated the tests again.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**